src/main/org/firebirdsql/gds/ng/wire/ProtocolCollection.java

```diff
@@ -96,7 +96,8 @@
         String[] protocolClasses = {
                 "org.firebirdsql.gds.ng.wire.version10.Version10Descriptor",
                 "org.firebirdsql.gds.ng.wire.version11.Version11Descriptor",
-                "org.firebirdsql.gds.ng.wire.version12.Version12Descriptor"
+                "org.firebirdsql.gds.ng.wire.version12.Version12Descriptor",
+                "org.firebirdsql.gds.ng.wire.version13.Version13Descriptor"
```

Comment:
This is not sufficient as this is only the fallback option, the class name also needs to be added to META-INF/services/org.firebirdsql.gds.ng.wire.ProtocolDescriptor